package.json

```diff
@@ -12,9 +12,11 @@
     "react:build": "react-scripts build",
     "electron:copy": "cp -r ./electron/. ./build",
     "dev": "concurrently -k \"BROWSER=none npm run react:start\" \"wait-on tcp:3000 && nodemon --exec electron electron/electron.js\"",
-    "build": "npm run clean && npm run react:build && npm run electron:copy",
-    "pack": "npm run build && electron-builder",
-    "release": "npm run build && electron-builder -mwl --publish=always",
+    "build": "run-s clean react:build electron:copy",
+    "release": "npm run pack -- -mwl --publish=always",
+    "pack": "run-s prepack electron-builder",
+    "prepack": "run-p build prepack:browsers",
+    "prepack:browsers": "node scripts/download-browsers",
```

Comment:
As @justinkambic experienced, this would still break for native deps, We can fix it by running the preparation and package script serially for all platforms and uploading the pack one by one.